packages/integrations/vue/src/index.ts

```diff
@@ -125,9 +126,11 @@ async function getViteConfiguration(
 
 	if (command === 'dev' && options?.devtools) {
 		const vueDevTools = (await import('vite-plugin-vue-devtools')).default;
+		const launchEditor =  typeof options.devtools === 'object' ? options.devtools.launchEditor : "code"
 		config.plugins?.push(
 			vueDevTools({
 				appendTo: VIRTUAL_MODULE_ID,
+				launchEditor
```

Comment:
I suggest we directly introduce into the whole ` VitePluginVueDevToolsOptions` (except ` appendTo `),  like:

```
	vueDevTools({
        ...devtoolsOptions,
				appendTo: VIRTUAL_MODULE_ID,
 })
```
This aligns with the plugin's own configuration, making it more intuitive for users and easier to extend.